cmd/sonobuoy/app/master.go

```diff
@@ -27,47 +28,57 @@ import (
 	"github.com/spf13/cobra"
 )
 
-var noExit bool
-var kubecfg Kubeconfig
+type aggregatorInput struct {
+	noExit  bool
+	kubecfg Kubeconfig
+}
```

Comment:
Just a single type/variable instead of 2 globals. I then create a single instance of this variable below in `NewCmdAggregator` which returns a closure around that instance.